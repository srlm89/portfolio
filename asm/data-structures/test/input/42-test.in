ERASE actual/42-test.out
INSERT 30 1.530423 Mr. & Mrs. Smith   720p
PRINT_IN_ORDER actual/42-test.out
INSERT 40 1.606426 Baby on Board (2009)    720p
INSERT 3 8.006707 One Missed Call   720p
REMOVE_ID 3
PRINT_IN_ORDER actual/42-test.out
INSERT 44 3.217073 The Last Samurai [2004] [720p]
INSERT 22 9.528289 Toy.Story.2.HD.720p.Hebrew.Dubbed.Also.English.x264-ALiS_20
INSERT 17 9.346551 Final Destination
REMOVE_ID 22
PRINT_IN_ORDER actual/42-test.out
INSERT 43 6.420979 Cellular.2004.HDTV.720p.AC3.x264-BTT
INSERT 4 0.010690 Eurotrip 720p
INSERT 2 4.486866 New York I Love You 720p   720p
INSERT 51 6.987177 Alien vs Predator 1-2  720p
REMOVE_ID 43
REMOVE_ID 40
PRINT_IN_ORDER actual/42-test.out
INSERT 16 6.609784 Liar Liar    720p
INSERT 1 3.009002 Madagascar 2 Escape 2 Africa [2008] [720p]
INSERT 14 2.031386 Basic Instinct 720p
INSERT 31 0.084415 Die Hard 1   720p
INSERT 19 4.504538 Memento    1080p
REMOVE_ID 17
REMOVE_ID 30
PRINT_IN_ORDER actual/42-test.out
INSERT 7 5.049597 Fast and the Furious 2 - 1080p
INSERT 57 8.692130 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS
INSERT 26 6.303452 Dragon Wars   720p
INSERT 54 1.114281 Bad Boy I [1995] [720p]
INSERT 59 2.487308 X-Men 2   1080p
INSERT 18 3.511779 Minority Report [720p]
REMOVE_ID 59
REMOVE_ID 16
REMOVE_ID 54
PRINT_IN_ORDER actual/42-test.out
INSERT 35 5.586206 Stardust   720p
INSERT 52 4.634384 Dumb and Dumber   720p
INSERT 42 6.260757 American Virgin 720P
INSERT 39 0.712379 Disturbia    720p
INSERT 47 2.834310 The Matrix   1080p
INSERT 8 1.057554 5_Centimeters_Per_Second_(2007)_[720p,BluRay,x264]_-_THORA
INSERT 0 7.780751 Knowing 2009 720p
REMOVE_ID 0
REMOVE_ID 51
REMOVE_ID 31
PRINT_IN_ORDER actual/42-test.out
INSERT 70 9.661781 Punisher 2   720p
INSERT 36 2.580189 Lord Of The Ring  - The Fellowship Of The Ring   720p
INSERT 60 0.460640 Starship Troopers 1  1080p
INSERT 65 7.848050 Defiance.1080p
INSERT 27 2.535609 James Bond 09 - The Man with the Golden Gun [1974]
INSERT 49 0.436011 Bourne Identity   720p
INSERT 63 9.439127 Rambo 4 [2008] [720p]
INSERT 64 3.791929 Interview with the Vampire   720p
REMOVE_ID 35
REMOVE_ID 70
REMOVE_ID 42
REMOVE_ID 57
PRINT_IN_ORDER actual/42-test.out
INSERT 48 5.577089 Blood River 720P
INSERT 41 3.884711 Charlie and the Chocolate Factory   720p
INSERT 72 1.584721 Breach 1080p
INSERT 45 7.553526 Madagascar 2 Escape 2 Africa [2008] [720p]
INSERT 81 4.912665 This Is It [2009] [720p]
INSERT 53 5.181329 Beauty.And.The.Beast.1991.720p.HDTV.x264-hV
INSERT 46 8.319987 Bangkok Dangerous 720p
INSERT 5 8.120162 Golden Compass   720p
INSERT 28 8.790510 X-Men 3    720p
REMOVE_ID 52
REMOVE_ID 19
REMOVE_ID 14
REMOVE_ID 27
PRINT_IN_ORDER actual/42-test.out
INSERT 25 1.432016 Body of Lies   720p
INSERT 6 5.807004 Get Smart (2008) 1080p
INSERT 86 0.529272 In The Electrical Mist   720p
INSERT 34 3.986528 Apocalypto   720p
INSERT 50 4.335438 The Bank Job [720p]
INSERT 38 4.480898 PBS
INSERT 76 3.910018 Street Fighter The Legend Of Chun Li 720P
INSERT 93 1.129380 Full.Metal.Jacket.REMASTERED.1987.720p.HDDVD.x264-SEPTiC
INSERT 71 8.664445 Fast and the Furious 2 - 1080p
INSERT 68 5.022465 This Is It [2009] [720p]
REMOVE_ID 60
REMOVE_ID 49
REMOVE_ID 47
REMOVE_ID 93
REMOVE_ID 28
PRINT_IN_ORDER actual/42-test.out
INSERT 74 0.786283 American Beauty   720p
INSERT 12 6.853198 Blade 3   720p
INSERT 21 1.967920 Daredevil - 720p
INSERT 91 8.603915 Batman Begins   1080p
INSERT 32 8.775754 Blade 3   720p
INSERT 83 8.630776 Braveheart 720p
INSERT 11 6.416775 The Matrix   1080p
INSERT 66 5.385415 Dr Dolite 3    720p
INSERT 73 6.142190 Lord Of The Ring  - The Two Tower   720p
INSERT 61 5.626262 Orphanage 720p
INSERT 84 8.427439 Around The World In 80 Days [2004] [720p]
REMOVE_ID 74
REMOVE_ID 11
REMOVE_ID 83
REMOVE_ID 91
REMOVE_ID 48
PRINT_IN_ORDER actual/42-test.out
INSERT 103 3.510726 The Extreme Movie 720P
INSERT 78 5.006600 G Force    720p
INSERT 102 4.340631 Bridge To Terabithia 720P
INSERT 110 4.189096 Pitch Black [720p]
INSERT 13 4.885982 Ratatouille [2007]
INSERT 20 4.426291 Final Destination 3   1080p
INSERT 56 5.516837 Wanted   1080p
INSERT 112 0.106334 American Pie Book Of Love   720p
INSERT 77 4.508201 Dr Dolite 3    720p
INSERT 9 6.296493 Fearless - 720p
INSERT 106 0.245568 American Beauty   720p
INSERT 55 1.000527 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS
REMOVE_ID 103
REMOVE_ID 86
REMOVE_ID 76
REMOVE_ID 106
REMOVE_ID 32
REMOVE_ID 61
PRINT_IN_ORDER actual/42-test.out
