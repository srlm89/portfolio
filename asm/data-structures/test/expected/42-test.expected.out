[ 30 1.530423 Mr. & Mrs. Smith   720p ]
[ 30 1.530423 Mr. & Mrs. Smith   720p ][ 40 1.606426 Baby on Board (2009)    720p ]
[ 17 9.346551 Final Destination ][ 30 1.530423 Mr. & Mrs. Smith   720p ][ 40 1.606426 Baby on Board (2009)    720p ][ 44 3.217073 The Last Samurai [2004] [720p] ]
[ 2 4.486866 New York I Love You 720p   720p ][ 4 0.010690 Eurotrip 720p ][ 17 9.346551 Final Destination ][ 30 1.530423 Mr. & Mrs. Smith   720p ][ 44 3.217073 The Last Samurai [2004] [720p] ][ 51 6.987177 Alien vs Predator 1-2  720p ]
[ 1 3.009002 Madagascar 2 Escape 2 Africa [2008] [720p] ][ 2 4.486866 New York I Love You 720p   720p ][ 4 0.010690 Eurotrip 720p ][ 14 2.031386 Basic Instinct 720p ][ 16 6.609784 Liar Liar    720p ][ 19 4.504538 Memento    1080p ][ 31 0.084415 Die Hard 1   720p ][ 44 3.217073 The Last Samurai [2004] [720p] ][ 51 6.987177 Alien vs Predator 1-2  720p ]
[ 1 3.009002 Madagascar 2 Escape 2 Africa [2008] [720p] ][ 2 4.486866 New York I Love You 720p   720p ][ 4 0.010690 Eurotrip 720p ][ 7 5.049597 Fast and the Furious 2 - 1080p ][ 14 2.031386 Basic Instinct 720p ][ 18 3.511779 Minority Report [720p] ][ 19 4.504538 Memento    1080p ][ 26 6.303452 Dragon Wars   720p ][ 31 0.084415 Die Hard 1   720p ][ 44 3.217073 The Last Samurai [2004] [720p] ][ 51 6.987177 Alien vs Predator 1-2  720p ][ 57 8.692130 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS ]
[ 1 3.009002 Madagascar 2 Escape 2 Africa [2008] [720p] ][ 2 4.486866 New York I Love You 720p   720p ][ 4 0.010690 Eurotrip 720p ][ 7 5.049597 Fast and the Furious 2 - 1080p ][ 8 1.057554 5_Centimeters_Per_Second_(2007)_[720p,BluRay,x264]_-_THORA ][ 14 2.031386 Basic Instinct 720p ][ 18 3.511779 Minority Report [720p] ][ 19 4.504538 Memento    1080p ][ 26 6.303452 Dragon Wars   720p ][ 35 5.586206 Stardust   720p ][ 39 0.712379 Disturbia    720p ][ 42 6.260757 American Virgin 720P ][ 44 3.217073 The Last Samurai [2004] [720p] ][ 47 2.834310 The Matrix   1080p ][ 52 4.634384 Dumb and Dumber   720p ][ 57 8.692130 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS ]
[ 1 3.009002 Madagascar 2 Escape 2 Africa [2008] [720p] ][ 2 4.486866 New York I Love You 720p   720p ][ 4 0.010690 Eurotrip 720p ][ 7 5.049597 Fast and the Furious 2 - 1080p ][ 8 1.057554 5_Centimeters_Per_Second_(2007)_[720p,BluRay,x264]_-_THORA ][ 14 2.031386 Basic Instinct 720p ][ 18 3.511779 Minority Report [720p] ][ 19 4.504538 Memento    1080p ][ 26 6.303452 Dragon Wars   720p ][ 27 2.535609 James Bond 09 - The Man with the Golden Gun [1974] ][ 36 2.580189 Lord Of The Ring  - The Fellowship Of The Ring   720p ][ 39 0.712379 Disturbia    720p ][ 44 3.217073 The Last Samurai [2004] [720p] ][ 47 2.834310 The Matrix   1080p ][ 49 0.436011 Bourne Identity   720p ][ 52 4.634384 Dumb and Dumber   720p ][ 60 0.460640 Starship Troopers 1  1080p ][ 63 9.439127 Rambo 4 [2008] [720p] ][ 64 3.791929 Interview with the Vampire   720p ][ 65 7.848050 Defiance.1080p ]
[ 1 3.009002 Madagascar 2 Escape 2 Africa [2008] [720p] ][ 2 4.486866 New York I Love You 720p   720p ][ 4 0.010690 Eurotrip 720p ][ 5 8.120162 Golden Compass   720p ][ 7 5.049597 Fast and the Furious 2 - 1080p ][ 8 1.057554 5_Centimeters_Per_Second_(2007)_[720p,BluRay,x264]_-_THORA ][ 18 3.511779 Minority Report [720p] ][ 26 6.303452 Dragon Wars   720p ][ 28 8.790510 X-Men 3    720p ][ 36 2.580189 Lord Of The Ring  - The Fellowship Of The Ring   720p ][ 39 0.712379 Disturbia    720p ][ 41 3.884711 Charlie and the Chocolate Factory   720p ][ 44 3.217073 The Last Samurai [2004] [720p] ][ 45 7.553526 Madagascar 2 Escape 2 Africa [2008] [720p] ][ 46 8.319987 Bangkok Dangerous 720p ][ 47 2.834310 The Matrix   1080p ][ 48 5.577089 Blood River 720P ][ 49 0.436011 Bourne Identity   720p ][ 53 5.181329 Beauty.And.The.Beast.1991.720p.HDTV.x264-hV ][ 60 0.460640 Starship Troopers 1  1080p ][ 63 9.439127 Rambo 4 [2008] [720p] ][ 64 3.791929 Interview with the Vampire   720p ][ 65 7.848050 Defiance.1080p ][ 72 1.584721 Breach 1080p ][ 81 4.912665 This Is It [2009] [720p] ]
[ 1 3.009002 Madagascar 2 Escape 2 Africa [2008] [720p] ][ 2 4.486866 New York I Love You 720p   720p ][ 4 0.010690 Eurotrip 720p ][ 5 8.120162 Golden Compass   720p ][ 6 5.807004 Get Smart (2008) 1080p ][ 7 5.049597 Fast and the Furious 2 - 1080p ][ 8 1.057554 5_Centimeters_Per_Second_(2007)_[720p,BluRay,x264]_-_THORA ][ 18 3.511779 Minority Report [720p] ][ 25 1.432016 Body of Lies   720p ][ 26 6.303452 Dragon Wars   720p ][ 34 3.986528 Apocalypto   720p ][ 36 2.580189 Lord Of The Ring  - The Fellowship Of The Ring   720p ][ 38 4.480898 PBS ][ 39 0.712379 Disturbia    720p ][ 41 3.884711 Charlie and the Chocolate Factory   720p ][ 44 3.217073 The Last Samurai [2004] [720p] ][ 45 7.553526 Madagascar 2 Escape 2 Africa [2008] [720p] ][ 46 8.319987 Bangkok Dangerous 720p ][ 48 5.577089 Blood River 720P ][ 50 4.335438 The Bank Job [720p] ][ 53 5.181329 Beauty.And.The.Beast.1991.720p.HDTV.x264-hV ][ 63 9.439127 Rambo 4 [2008] [720p] ][ 64 3.791929 Interview with the Vampire   720p ][ 65 7.848050 Defiance.1080p ][ 68 5.022465 This Is It [2009] [720p] ][ 71 8.664445 Fast and the Furious 2 - 1080p ][ 72 1.584721 Breach 1080p ][ 76 3.910018 Street Fighter The Legend Of Chun Li 720P ][ 81 4.912665 This Is It [2009] [720p] ][ 86 0.529272 In The Electrical Mist   720p ]
[ 1 3.009002 Madagascar 2 Escape 2 Africa [2008] [720p] ][ 2 4.486866 New York I Love You 720p   720p ][ 4 0.010690 Eurotrip 720p ][ 5 8.120162 Golden Compass   720p ][ 6 5.807004 Get Smart (2008) 1080p ][ 7 5.049597 Fast and the Furious 2 - 1080p ][ 8 1.057554 5_Centimeters_Per_Second_(2007)_[720p,BluRay,x264]_-_THORA ][ 12 6.853198 Blade 3   720p ][ 18 3.511779 Minority Report [720p] ][ 21 1.967920 Daredevil - 720p ][ 25 1.432016 Body of Lies   720p ][ 26 6.303452 Dragon Wars   720p ][ 32 8.775754 Blade 3   720p ][ 34 3.986528 Apocalypto   720p ][ 36 2.580189 Lord Of The Ring  - The Fellowship Of The Ring   720p ][ 38 4.480898 PBS ][ 39 0.712379 Disturbia    720p ][ 41 3.884711 Charlie and the Chocolate Factory   720p ][ 44 3.217073 The Last Samurai [2004] [720p] ][ 45 7.553526 Madagascar 2 Escape 2 Africa [2008] [720p] ][ 46 8.319987 Bangkok Dangerous 720p ][ 50 4.335438 The Bank Job [720p] ][ 53 5.181329 Beauty.And.The.Beast.1991.720p.HDTV.x264-hV ][ 61 5.626262 Orphanage 720p ][ 63 9.439127 Rambo 4 [2008] [720p] ][ 64 3.791929 Interview with the Vampire   720p ][ 65 7.848050 Defiance.1080p ][ 66 5.385415 Dr Dolite 3    720p ][ 68 5.022465 This Is It [2009] [720p] ][ 71 8.664445 Fast and the Furious 2 - 1080p ][ 72 1.584721 Breach 1080p ][ 73 6.142190 Lord Of The Ring  - The Two Tower   720p ][ 76 3.910018 Street Fighter The Legend Of Chun Li 720P ][ 81 4.912665 This Is It [2009] [720p] ][ 84 8.427439 Around The World In 80 Days [2004] [720p] ][ 86 0.529272 In The Electrical Mist   720p ]
[ 1 3.009002 Madagascar 2 Escape 2 Africa [2008] [720p] ][ 2 4.486866 New York I Love You 720p   720p ][ 4 0.010690 Eurotrip 720p ][ 5 8.120162 Golden Compass   720p ][ 6 5.807004 Get Smart (2008) 1080p ][ 7 5.049597 Fast and the Furious 2 - 1080p ][ 8 1.057554 5_Centimeters_Per_Second_(2007)_[720p,BluRay,x264]_-_THORA ][ 9 6.296493 Fearless - 720p ][ 12 6.853198 Blade 3   720p ][ 13 4.885982 Ratatouille [2007] ][ 18 3.511779 Minority Report [720p] ][ 20 4.426291 Final Destination 3   1080p ][ 21 1.967920 Daredevil - 720p ][ 25 1.432016 Body of Lies   720p ][ 26 6.303452 Dragon Wars   720p ][ 34 3.986528 Apocalypto   720p ][ 36 2.580189 Lord Of The Ring  - The Fellowship Of The Ring   720p ][ 38 4.480898 PBS ][ 39 0.712379 Disturbia    720p ][ 41 3.884711 Charlie and the Chocolate Factory   720p ][ 44 3.217073 The Last Samurai [2004] [720p] ][ 45 7.553526 Madagascar 2 Escape 2 Africa [2008] [720p] ][ 46 8.319987 Bangkok Dangerous 720p ][ 50 4.335438 The Bank Job [720p] ][ 53 5.181329 Beauty.And.The.Beast.1991.720p.HDTV.x264-hV ][ 55 1.000527 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS ][ 56 5.516837 Wanted   1080p ][ 63 9.439127 Rambo 4 [2008] [720p] ][ 64 3.791929 Interview with the Vampire   720p ][ 65 7.848050 Defiance.1080p ][ 66 5.385415 Dr Dolite 3    720p ][ 68 5.022465 This Is It [2009] [720p] ][ 71 8.664445 Fast and the Furious 2 - 1080p ][ 72 1.584721 Breach 1080p ][ 73 6.142190 Lord Of The Ring  - The Two Tower   720p ][ 77 4.508201 Dr Dolite 3    720p ][ 78 5.006600 G Force    720p ][ 81 4.912665 This Is It [2009] [720p] ][ 84 8.427439 Around The World In 80 Days [2004] [720p] ][ 102 4.340631 Bridge To Terabithia 720P ][ 110 4.189096 Pitch Black [720p] ][ 112 0.106334 American Pie Book Of Love   720p ]
